### DISCUSSION TITLE
Dumb question about unknown games

### DISCUSSION THREAD

**Discussion by fabriciotm** [sanni/cartreader#796] [Need help]
Sorry for this dumb question, but I searched the forum and can't found the answer.
How the reader works when he don't found the game in database?
I have some modified Atari and Coleco games modified in the 80's. They are rare games and needed to be dumped. 
How the reader will identify the game? There's a option to manually select the bankswitch or game name? Some of then are only rebranded games and don't use any special chip or exotic bankswitch scheme, but the CRC is diferent.

**Comment by Ancyker** [comment]
It depends on the game system. I am not familiar with Atari or Coleco games myself, but for systems like the NES you will have to manually tell the cart reader how to dump it if the game is not detected because NES games do not contain any data on how the chips are laid out. 

But for games that do, the game can still be dumped fine. It's just the cart reader cannot verify it or potentially even name it correctly as some systems do not put the game's name in the header (or some have a header with a name but the name is always the same, i.e. SMS games are always "TMR SEGA" regardless of the game). If the systems you are working with fall into this category they will still automatically dump fine but if there is no name in the header you'll get a file with a placeholder name instead of the name of the game. 

Regardless of which of the above applies, the cart reader will say verification failed because the game isn't known to it so it could not verify it dumped correctly, but if you know what the CRC is supposed to be you can verify it manually.

**Reply by Ancyker** [reply]
Also, if you don't know what the CRC is supposed to be and you cannot find it online, your only choice is to dump the game several times, preferably on more than 1 cart reader, and see if the CRC changes. If the CRC does not change, try playing the dumped ROM to see if it works (preferably to 100% completion). If it does, the game probably dumped fine. Ideally, in the case where a CRC is not known, we'd want to dump the game from multiple cartridges using multiple different cart readers to confirm we have a good/clean dump.
